package.json

```diff
@@ -31,9 +31,10 @@
   "dependencies": {
     "arguejs": "^0.2.3",
     "lodash": "^4.15.0",
+    "long": "^3.2.0",
     "nan": "^2.0.0",
     "node-pre-gyp": "^0.6.0",
-    "protobufjs": "^5.0.0"
+    "protobufjs": "https://github.com/dcodeIO/protobuf.js/tarball/ef43acff5"
```

Comment:
Waiting on @dcodeIO to publish a new release. Then it can be changed to `^6.2.0` or so.